### DISCUSSION TITLE
About Gtk.FileDialog method bindings

### DISCUSSION THREAD

**Discussion by PlatinumLucario** [gircore/gir.core#901] [Q&A] [answered]
After releasing the 0.4.0 milestone, I'm been really curious. Are the methods for ``open``, ``open_multiple``, ``save``, ``select_folder`` and ``select_multiple_folders`` are not available for language bindings? Because I don't see them in there, but they're in there for Gtk.Internal.FileDialog though.

The official GTK docs for FileDialog don't mention about them not being available to the language bindings, unless I've missed something in there.

Do I need to use it in a different way? Or do I need to manually use them via the Gtk.Internal ones?

**Comment by fsobolev** [comment, marked as answer]
There's an issue #900 about that.
The old method of using marshalling still works. https://github.com/NickvisionApps/Application/blob/c2bb771d481b724d1f3edca59b6de2b40c03e25f/NickvisionApplication.GNOME/Views/MainWindow.cs#L215

**Reply by PlatinumLucario** [reply]
Ah, that explains it, and yeah, I'll continue using the old method

**Reply by fsobolev** [reply]
@PlatinumLucario until needed stuff is implemented in gir.core, you can try GirExt https://github.com/NickvisionApps/GirExt
